### ISSUE TITLE
Git conventions

### ISSUE THREAD

**Issue by object-Object** [walksanatora/hexserver3mods#7]
I'd suggest either working on separate branches and only merging to master when we want to update the pack, and/or setting up the Pages workflow to only run in certain cases; eg. a new tag is added to `master`, a commit containing `[Release]` is pushed to master, or the version number in `pack.toml` is bumped. Also, ideally we should only be merging to master via pull requests.

**Comment by object-Object** [comment]
(paging @walksanatora)

**Comment by walksanatora** [comment]
oh yes
once we actually start there will be a sepereate branch
(since currently the packwiz only reloads from `master`)

**Comment by walksanatora** [comment]
(also you would have to help me with setting up a action like that as the only expierennce I have with actions is running files)

**Comment by LlamaSageR** [comment]
Sounds like a plan. For now we can probably just update the master branch since it isn't finalized, but once it is launched we can have branches to begin updates and whatnot. I lack experience with Github though.

**Comment by object-Object** [comment]
I've enabled branch protection so pushes to `master` need to go through a PR with at least one approval. Also added a `dev` branch where we should be able to stage most of our changes — technically it's also protected but only to prevent deleting it. Basically, work as you normally would on the `dev` branch, and we'll do PRs from there to `master` when we want to release things to the public.

**Comment by object-Object** [comment]
You can of course also make additional branches and PR from there to `dev` etc.

**Comment by walksanatora** [comment]
yep i just made a PR for adding YTTR to dev (it looks interesting and usefull, and even petra likes it)
